BASE/src/Microsoft.ApplicationInsights/Channel/Transmission.cs

```diff
@@ -133,6 +162,18 @@ public ICollection<ITelemetry> TelemetryItems
             get; private set;
         }
 
+        /// <summary>
+        /// Send signal to Flush Async.
+        /// </summary>
+        public void SetFlushTaskCompletionSource(bool manualFlushAsyncFlag)
```

Comment:
Why have a bool parameter when the only valid value is `true`?
I see there are places in this PR where we call `SetFlushTaskCompletionSource(false)`. What does that do?